packages/api/src/graphql/collection.ts

```diff
@@ -35,17 +36,48 @@ const COLLECTION_EXIST = gql`
 
 const COLLECTION_LIST = gql`
   query CollectionList($databaseName: String!) {
-    collectionList(databaseName: $databaseName)
+    collectionList(databaseName: $databaseName) {
+      name
+      indexes
+      schema {
+        name
+        kind
+        indexed
+      }
+      metadata {
```

Comment:
Metada is ambiguous, should he ownership instead

Reply:
done